src/sentry/reprocessing2.py

```diff
@@ -433,8 +539,6 @@ def pop_remaining_event_ids_from_redis(key):
 
         event_ids_batch.append(event_id)
 
-    client.delete(key)
```

Comment:
I believe this line is missing. Since we expire keys it's not a real leak, but with high traffic we might run out of space temporarily.

I believe you wanted to reuse `pop_remaining_event_is_from_redis` (did I really write that sort of function name?) and then forgot, and that's why delete is missing.

Reply:
ah okay, i believe the original `buffered_remaining_...` implementation might have the same issue in that that case, as it does not delete the original keys for the batch: it deletes the temporary keys created for the task. if that assertion is correct, and we do want to delete all of the keys associated with a batch in redis once we've processed them, there are two things that need to be done:
- undo the change related to this helper
- patch up `buffered_remaining_events` to delete all their redis keys associated with a batch in a different PR

thoughts?

Reply:
edit: rename deletes the old key, so this is just missing the delete. i'll revert the change.